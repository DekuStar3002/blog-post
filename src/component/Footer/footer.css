.footer {
  background-color: black;
  color: whitesmoke;
  display: flex;
  align-items: center;
  justify-content: flex-start;
  position: absolute;
  bottom: 0;
  height: 10vh;
  width: 100%;
}

.footer p {
  margin: 0 5vw;
}