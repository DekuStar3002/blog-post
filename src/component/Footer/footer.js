import React from 'react';
import './footer.css';
const Footer = () => {
  const year = new Date().getFullYear();
  return (
    <div className='footer'>
      <p>@artifact.com {year}</p>
    </div>
  )
}

export default Footer;