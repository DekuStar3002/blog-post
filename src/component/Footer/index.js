import React from 'react';
import './Footer.css';
const Footer = () => {
  const year = new Date().getFullYear();
  return (
    <div className='footer'>
      <p>@artifact.com {year}</p>
    </div>
  )
}

export default Footer;