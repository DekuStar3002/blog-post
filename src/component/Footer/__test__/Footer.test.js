import Footer from '..';
import { render } from "@testing-library/react";

describe('Footer', () => { 
  it('should render correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  })
});