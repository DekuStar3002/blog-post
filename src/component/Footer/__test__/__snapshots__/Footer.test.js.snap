// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p>
      @artifact.com 2023
    </p>
  </div>
</DocumentFragment>
`;
